src/array/cpu/spmm.h

```diff
@@ -143,7 +144,8 @@ void SpMMSumCsr(const BcastOff& bcast, const CSRMatrix& csr, NDArray ufeat,
 #ifdef USE_LIBXSMM
   const bool no_libxsmm =
        bcast.use_bcast || std::is_same<DType, double>::value;
-  if (!no_libxsmm) {
+  const bool use_libxsmm = globalContext().libxsmm();
```

Comment:
You can as well merge `use_libxsmm` into `no_libxsmm`.  I assume we don't need two variables.

Reply:
Make sense, combine two variables.